config/routes.rb

```diff
@@ -6,7 +6,12 @@
 
   get 'home/index'
 
-  resources :users, only: [:index]
+  as :admin do
+    get  'volunteers',  to: 'users#new'
+    post 'volunteers',  to: 'users#add_user'
+  end
```

Comment:
Here 'volunteers' is arbitrary. It can be anything but not user or users, because devise gem is reserving these for itself